backend/src/pong/game/game.spec.ts

```diff
@@ -31,4 +32,61 @@ describe('Game unit tests', () => {
 		expect(service).toBeDefined();
 		expect(controller).toBeDefined();
 	});
+
+	describe('CreateGameDto', () => {
+		const validator = new ValidationPipe({
+			whitelist: true,
+			forbidNonWhitelisted: true,
+			transform: true,
+		});
+
+		let testObject = {
+			player_one: 1,
+			player_two: 1,
+		};
+
+		const meta: ArgumentMetadata = {
+			type: 'body',
+			metatype: CreateGameDto,
+		};
+
+		it('should throw an error when player 1 & 2 have the same id', async () => {
+			await expect(validator.transform(testObject, meta)).rejects.toThrow(
+				'Bad Request',
+			);
+		});
+
+		it('should throw error when player_one is empty', async () => {
+			testObject.player_one = null;
+			await expect(validator.transform(testObject, meta)).rejects.toThrow(
+				'Bad Request',
+			);
+		});
+
+		it('should throw error when player_two is empty', async () => {
+			testObject.player_one = 1;
+			testObject.player_two = null;
+			await expect(validator.transform(testObject, meta)).rejects.toThrow(
+				'Bad Request',
+			);
+		});
+	});
+
+	describe('updateGameDto', () => {
+		it('should not throw an error when update object is empty', async () => {
+			const validator = new ValidationPipe({
+				whitelist: true,
+				forbidNonWhitelisted: true,
+				transform: true,
```

Comment:
change to use `globalValidationPipeOptions` after pulling updated main (after merging #51)